compiler/query_structure.rs

```diff
@@ -131,3 +109,36 @@ fn extract_query_structure_from_branch(
         }
     })
 }
+
+fn extend_labels_from<'a>(
+    resolved_labels: &mut HashMap<Label, Type>,
+    vertex_annotations: impl Iterator<Item = (&'a Vertex<Variable>, &'a Arc<BTreeSet<answer::Type>>)>,
+) {
+    resolved_labels.extend(vertex_annotations.filter_map(|(vertex, type_)| {
+        match (vertex.as_label(), type_.iter().exactly_one()) {
+            (Some(label), Ok(type_)) => Some((label.clone(), type_.clone())),
+            _ => None,
+        }
+    }));
+}
+
+fn extend_function_text_from(
+    conjunction: &Conjunction,
+    function_texts: &mut HashMap<Constraint<Variable>, String>,
```

Comment:
Can we specify these are the `function_calls_syntax` perhaps? Im writing the schema retrieval code and been using `syntax` for the TypeQL representations. It won't help but you could use `call_syntax` instead here or?